.github/workflows/build.yml

```diff
@@ -0,0 +1,92 @@
+---
+
+name: Build wheels
+on: push
+jobs:
+  manylinux_wheels:
+    name: Python ${{ matrix.python_tag }} ${{ matrix.platform }} wheels
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform:
+          - manylinux1_x86_64
+          - manylinux2014_x86_64
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Docker
+        run: sudo apt-get install -y --no-install-recommends docker
+      - name: Pull Docker image
+        run: |
+          DOCKER_IMAGE="quay.io/pypa/${{ matrix.platform }}"
+          echo "::set-env name=DOCKER_IMAGE::$DOCKER_IMAGE"
+          docker image pull "$DOCKER_IMAGE"
+      - name: Build manylinux wheels
+        env:
+          PYTHON_TAGS: 'cp35-cp35m cp36-cp36m cp37-cp37m cp38-cp38'
+        run: |
+          docker container run \
+            --rm \
+            -e PLAT=${{ matrix.platform }} \
+            -e PYTHON_TAGS="$PYTHON_TAGS" \
+            -v "$(pwd)":/io \
+            --workdir '/io' \
+            "$DOCKER_IMAGE" \
+            /io/packaging/build/manylinux.sh
+      - uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: dist
+  macosx_wheels:
+    name: Python ${{ matrix.python_version }} MacOS wheels
+    # Note: the container MacOS version may differ from the SDK
+    # used to build Python there.  It is the latter that determines
+    # the wheel's platform tag.
+    # https://github.com/actions/virtual-environments/issues/696
+    #
+    # See also:
+    # https://github.com/actions/virtual-environments/blob/master/images/macos/macos-10.15-Readme.md
+    # for environment description.
```

Comment:
It's usually preferable to use Python from python.org to achieve the best interoperability of the resulting platform-specific wheels: 
https://github.com/ansible/pylibssh/blob/8463696/.github/workflows/build-test-n-publish.yml#L177-L247

Reply:
I'm interested in learning more about why this is because I don't recall having ever seen the `.pkg` being grabbed and installed to build MacOS wheels. I suppose it does seem like it couldn't hurt, but are there specific examples you had in mind of where you've otherwise seen interoperability suffer?

Reply:
Here's old research on this topic: https://github.com/MacPython/wiki/wiki/Spinning-wheels.
TL;DR if you rely on something somebody else built (or even you, in case of using pyenv, for example), your Python may be lacking some modules (they can get skipped) and the system API it's built against can be too high. Also, for older Python versions they ship dual-releases with both 32- and 64-bit binaries (it's not as important today, though).
But I've seen a lot of projects using this approach even back in the days of Travis. I've also found, it's one of the most performant methods of getting Python you want (compared to using pyenv or brew that are often ridiculously slow).

Reply:
cibuildwheel does this for you, as well.

Reply:
Yep, but my snippet is a bit more clever AFAIK: it automatically determines the latest patch version available for a given minor version.